rules/integrations/cloud_defend/container_workload_protection.toml

```
@@ -0,0 +1,31 @@
+[metadata]
+creation_date = "2023/03/29"
+integration = ["cloud_defend"]
+maturity = "production"
+min_stack_comments = "Initial version of cloud_defend alerts rule"
```

```suggestion
min_stack_comments = "Initial version of the Container Workload Protection alerts"
```
Maybe we should point this to the feature? Instead of the rule itself